lib/basedriver/device-settings.js

```diff
@@ -25,6 +25,14 @@ const GLOBAL_DEFAULT_SETTINGS = {
   //      if a user use `width=750 × height=1334` pixels's base template image.
   fixImageTemplateScale: false,
 
+  // Users might have scaled template image to reduce their storage size.
+  // This setting allows users to scale a template image they send to Appium server
+  // so that Appium server compare actual scale users orginaly have.
+  // e.g. A user have `270 × 32 pixels` image oridinally `1080 × 126 pixels`
```

Comment:
originally

Reply:
If a user has an image of 270 x 32 pixels which was originally 1080 x 126 pixels